### PR TITLE
Add comment count to the communications tab.

### LINKED ISSUE #3629: Show how many comments there are on the communication tab

### PR DESCRIPTION
**Pull request by frjo**
Fixes #3629

### CODE REVIEW

File: hypha/apply/funds/templates/funds/applicationsubmission_detail.html [hunk 1]
```diff
@@ -46,7 +46,7 @@ <h1 class="mb-0 mt-2 font-medium">{{ object.title }}</h1>
                     </a>
 
                     <a class="tab__item" href="#communications" data-tab="tab-2">
-                        {% trans "Communications" %}
+                        {% trans "Communications" %} ({{ object.activities.comments.count }})
```
Comment (theskumar) [inline hypha/apply/funds/templates/funds/applicationsubmission_detail.html]: I belive this should be the count of visible comment, no all the comments. I see have a submission with only one comment but the count is 2. 

Screenshot 2023-10-27 at 9  48 05@2x

**Review by theskumar — CHANGES_REQUESTED** [review]

File: hypha/apply/funds/templates/funds/applicationsubmission_detail.html (hunk 1 above)
Comment (frjo) [reply hypha/apply/funds/templates/funds/applicationsubmission_detail.html]: Good catch. Need to filter the count on what comments the user can see. Or we only show it to staff.

**Comment by frjo** [comment]
@theskumar Really simple fixes. Managed to forget that we already have "comments" available in this context.

**Review by theskumar — APPROVED** [review]
Nice 🎉

**Pull request merged**